### PR TITLE
Reactivate Tests and update to php8.3 

### PR DESCRIPTION
**Pull request by m-timmermann**
This PR:
* Update to php 8.3
* Update phpunit to 11
* Reactivate the tests
* Changed the group annotations to attributes.
* changed the namespacing to match composer 
  * this should also fix most of the problems regarding executing test local or via IDE
  
Issue [4870](https://github.com/meplato/store.v3/issues/4870)

### CODE REVIEW

File: tests/HttpClientTest.php [hunk 1]
```diff
@@ -12,22 +12,22 @@
...
-	 * @group root
-	 * @group http
 	 */
+	#[Group('root')]
+	#[Group('http')]
 	public function testNewHttpClient()
 	{
 		$client = new HttpClient();
-		$resp = $client->execute("GET", "https://store2.meplato.com/");
+		$resp = $client->execute("GET", "https://store.meplato.com/");
```
Comment (m-timmermann) [inline tests/HttpClientTest.php:30]: This test is failing when the store is down/in maintenance
I do not know how to feel about that.

File: tests/HttpClientTest.php [hunk 2]
```diff
@@ -12,18 +12,18 @@
...
 {
 	/**
 	 * Test if the HTTP client works, in general.
-	 *
-	 * @group root
-	 * @group http
 	 */
+	#[Group('root')]
+	#[Group('http')]
 	public function testNewHttpClient()
```
Comment (m-timmermann) [inline tests/HttpClientTest.php:27]: This test fails when store is in maintenance mode.
Do not know how to feel about that.

**Review by iavorJ — APPROVED** [review]
LGTM, thank you! Tested locally as well

Super nit - maybe we can update the README about the syntax change for groups, as it mentions it (from `@group x` to `#[Group('x')]`)

File: tests/HttpClientTest.php (hunk 2 above)
Comment (iavorJ) [reply tests/HttpClientTest.php:27]: I think that's fine, even though it has kind of an "integration" test feel to it. While on the subject though - do we still need the store2 subdomain?

Comment (XoseRamon) [reply tests/HttpClientTest.php:27]: `store2` refers to the API (`api/v2`), not the application.

**Pull request merged**